### PR TITLE
chore(i18n,curriculum): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

**Review by texas2010 — DISMISSED** [review]

**Review by ojeytonwilliams — DISMISSED** [review]

File: curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Tal como o operador de igualdade, o operador maior que converterá os tipos de d
 '1' >  9
 ```
 
-Em ordem, essas expressões seriam iguais à `true`, `false`, `true` e `true`.
+Em ordem, essas expressões seriam iguais à true, `true`, `true`, `false` e `false`.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: Does this not still look incorrect?

Comment (ieahleen) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: Yes I would expect it to be
```suggestion
Em ordem, essas expressões seriam iguais à `true`, `true`, `false` e `false`.
```

now the values in backticks match, but it seems there is an extra one before outside of backticks

Comment (nhcarrigan) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: Yep that's incorrect.

Comment (nhcarrigan) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: @ShaunSHamilton I approved your fix over on Crowdin.

Comment (ieahleen) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: left a message on the Portuguese chat room, I don't know if we have Portuguese proofreaders on GitHub to tag as needed

Comment (ieahleen) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: Or that was faster

Comment (ShaunSHamilton) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: @nhcarrigan Do we merge this anyway? Or, will Dr. Camperbot rebase with an update?

Comment (nhcarrigan) [reply curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md:25]: Camperbot will rebase. I'll go ahead and re-run the action :)

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Review by naomi-lgbt — APPROVED** [review]
🚢 it

**Pull request merged**